### PR TITLE
Basic styling

### PR DESCRIPTION
**Pull request by nicoeg**
Courtesy of @peterbork

image

### CODE REVIEW

File: lib/csgo_stats_web/templates/match/show.html.leex [hunk 1]
```diff
@@ -1,41 +1,97 @@
...
-<div style="display: flex;justify-content: space-between;">
-  <div>
-    <%= Map.from_struct(@match) |> Map.drop([:players, :kill_feed]) |> Jason.encode!(pretty: true) %>
+<div class="game-info">
+  <div class="stat">
+    <div class="key">
+      Map
+    </div>
+    <div class="value">
+      Cache
```
Comment (trarbr) [inline lib/csgo_stats_web/templates/match/show.html.leex]:
```suggestion
      <%= @match.map %>
```

File: lib/csgo_stats_web/templates/match/show.html.leex [hunk 2]
```diff
@@ -1,41 +1,97 @@
...
+        </div>
+        <%= for {username, player} <- @match.players, player.team == :ct do %>
+        <div class="table-row">
+          <div class="table-data"><%= username %></div>
+          <div class="table-data"><%= player.kills %></div>
+          <div class="table-data"><%= player.assists %></div>
+          <div class="table-data"><%= player.deaths %></div>
+        </div>
+        <% end %>
+        <%= if @match.players |> Enum.filter(fn {_, player} -> player.team == :ct end) |> Enum.empty?() do %>
```
Comment (trarbr) [inline lib/csgo_stats_web/templates/match/show.html.leex]: This should also be applied for the T's

```suggestion
        <%= unless Enum.any?(@match.players, fn {_, player} -> player.team == :ct end) do %>
```

**Review by trarbr — COMMENTED** [review]
That looks great, thanks @peterbork (and @nicoeg )

90% of it even looks good on my phone 😉: 

Screenshot_20191222-170845

I think the table for CT's and T's can be extracted into a function - I think the only things that vary are: the class for wrapping div, the icon (which could be specified in css based on wrapping class?),  the team name, and the score.  Wanna tinker with that? Or we can leave it for a round of refactoring. In fact, I'm not sure how to extract bits from leex templates.

Also, the .leex files could use some manual formatting (e.g. space around brackets https://github.com/trarbr/csgo_stats/blob/394949ee68d7f9f0905abb1bd6cecd7a71ac9253/lib/csgo_stats_web/templates/match/show.html.leex#L12)

**Comment by nicoeg** [comment]
> That looks great, thanks @peterbork (and @nicoeg )
> 
> 90% of it even looks good on my phone 😉:
> 
> I think the table for CT's and T's can be extracted into a function - I think the only things that vary are: the class for wrapping div, the icon (which could be specified in css based on wrapping class?), the team name, and the score. Wanna tinker with that? Or we can leave it for a round of refactoring. In fact, I'm not sure how to extract bits from leex templates.

Didn't actually test mobile, forgot all about it :D - Just made some minor adjustments to make it even more viewable on mobile. 
image

I will wait with the extraction of the templates because I would actually like to change the semantics so that it's possible to move the score on top of the table when on smaller screens. This will give us more room for data in the table when we add that in the future.
I've created an issue for this #11

File: lib/csgo_stats_web/templates/match/show.html.leex (hunk 2 above)
Comment (nicoeg) [reply lib/csgo_stats_web/templates/match/show.html.leex]: Fixxed

File: lib/csgo_stats_web/templates/match/show.html.leex (hunk 1 above)
Comment (nicoeg) [reply lib/csgo_stats_web/templates/match/show.html.leex]: Fixxed

**Review by trarbr — APPROVED** [review]
Fantastic 💜

**Pull request merged**